README.md

```diff
@@ -1,2 +1,27 @@
-# factory-trytond
-Factory Boy - Trytond integration
+# Factory-trytond
+
+Factory-trytond is a [factory_boy](https://factoryboy.readthedocs.io/en/latest/introduction.html) extension developed to work with [Tryton ERP](https://www.tryton.org/). We can create our own ERP's model factories to do some testing or to populate our databases with some sample data.
```

Comment:
The same, would you please split this long paragraph in multiple shorter consecutive lines? Markdown will render all the same but maintaining the documentation source will be nicer